scala/scala.bzl

```diff
@@ -661,9 +649,9 @@ scala_test = rule(
   attrs={
       "main_class": attr.string(default="org.scalatest.tools.Runner"),
       "suites": attr.string_list(),
-      "_scalatest": attr.label(default=Label('//external:io_bazel_rules_scala/dependency/scalatest/scalatest'), allow_files=True),
+      "_scalatest": attr.label(default=Label("//external:io_bazel_rules_scala/dependency/scalatest/scalatest"), allow_files=True),
```

Comment:
It was only this one. The _scalatest was the only label that used double quotes so it originally exposed an issue where our repo conversion script wasn't finding this string. Had put it in here originally so could do some internal tests but can do that in a separate PR as well.